webscrambles/src/main/kotlin/org/worldcubeassociation/tnoodle/server/webscrambles/pdf/GeneralScrambleSheet.kt

```diff
@@ -33,32 +32,27 @@ class GeneralScrambleSheet(scrambleSet: ScrambleSet, activityCode: ActivityCode)
         val maxScrambleImageHeight = (allScramblesHeight / scramblesPerPage - 2 * SCRAMBLE_IMAGE_PADDING).toInt()
 
         // We don't let scramble images take up too much of a the page
-        val maxScrambleImageWidth = (availableWidth / 2).toInt().takeUnless {
-            // TODO - If we allow the megaminx image to be too wide, the
-            //   Megaminx scrambles get really tiny. This tweak allocates
-            //   a more optimal amount of space to the scrambles. This is possible
-            //   because the scrambles are so uniformly sized.
-            scramblingPuzzle.shortName == "minx"
-        } ?: 190
+        val maxScrambleImageWidth = (availableWidth / 3).toInt()
 
         val scrambleImageSize = scramblingPuzzle.getPreferredSize(maxScrambleImageWidth, maxScrambleImageHeight)
 
+        val allScrambleStrings = scrambleSet.allScrambles.toPDFStrings(scramblingPuzzle.shortName)
+
         val scrambleImageHeight = scrambleImageSize.height.toFloat()
         val scrambleColumnWidth = availableWidth - indexColumnWidth - scrambleImageSize.width
 
-        val allScrambleStrings = scrambleSet.allScrambles.toPDFStrings(scramblingPuzzle.shortName)
+        val availableArea = Rectangle(scrambleColumnWidth, scrambleImageHeight - 2 * SCRAMBLE_MARGIN)
```

Comment:
I see that you are subtracting the vertical margin here that was previously subtracted within the `getFontConfiguration` method.

If you look into the previous (before your changes in this PR) version of `getFontConfiguration`, you will see there are a lot of other margins and paddings being thrown around as well (`SCRAMBLE_PADDING_VERTICAL_TOP`, `SCRAMBLE_PADDING_VERTICAL_BOTTOM`, and others). What happened to them?

Reply:
I'll check on the duplication of removing stuffs. I think we were using too much constants for padding so I replaced them with 1. I'm OK with 2 (vertical and horizintal).